reader.lua

```diff
@@ -205,6 +205,9 @@ end
 
 local UIManager = require("ui/uimanager")
 
+-- Apply developer patches
+require("livepatch")
+
```

Comment:
You know more than us about what kind of stuff can be in your scripts.always/afterupdate, and if they are fine being executed that late.
But I feel the Android patch.lua might be used to tweak the framebuffer dimensions (@pazos ? @NiLuJe ) , and I guess it needs to be run before reader.lua - or may be at its very top. So, you might need to stage in live.patch.lua, that you could import early and run the 0* scripts early, and others later ?
Or `local livepatch = require("livepatch")` at top and have it run any existing patch.lua (without moving it) at require time - and for the later stuff, here do livepatch.executeLivePatches()